packages/ui/src/stores/studyBuilder.ts

```diff
@@ -53,15 +50,16 @@ export const useStudyBuilderStore = defineStore("studyBuilder", () => {
   function routeStudyId() {
     if (route.name === "study") {
```

Comment:
might be cleaner to make this `if (route.name !== "study") return "";` if possible to get the same semantics